ffi/capi/src/utf.rs

```diff
@@ -138,6 +138,7 @@ impl TryWriteable for PotentiallyInvalidUtf16<'_> {
 mod test {
     #![allow(invalid_from_utf8)] // only way to construct the error
     use super::*;
+    use alloc::format;
```

Comment:
This one was showing up as a compile failure.

Reply:
I believe this is not the right fix, the crate should not be `no_std` if `cfg(test)`, like all other crates.

Reply:
Yep. Looks like there were 2 crates with the wrong `no_std` declaration. Fixed!